diff --git a/lib/multi-semantic-release.js b/lib/multi-semantic-release.js
index 1247e618244dc35f6d60d1e484fbd1a84e504b2a..4b6f1bb5aae5219db2069412f56094af0e33a664 100644
--- a/lib/multi-semantic-release.js
+++ b/lib/multi-semantic-release.js
@@ -114,6 +114,9 @@ async function releasePackage(package_, createInlinePlugin, multiContext, flags)
     options.ci = flags.ci === undefined ? options.ci : flags.ci;
     options.branches = flags.branches ? castArray(flags.branches) : options.branches;
 
+    // Patching so plugins from globalOptions are also inherited.
+    options.plugins = [...(multiContext.globalOptions.plugins || []), ...(options.plugins || [])]
+
     // This options are needed for plugins that do not rely on `pluginOptions` and extract them independently.
     options._pkgOptions = packageOptions;

